examples/model_hosting/notebook/simple_scheduled_calculation/SimpleScheduledCalculation.ipynb

```diff
@@ -480,7 +480,7 @@
     "    ),\n",
     "})\n",
     "data_fetcher = DataFetcher(data_spec)\n",
-    "data_fetcher.time_series.fetch_dataframe([\"x\", \"y\", \"mean_x_y\"]).plot(x=\"timestamp\")"
+    "data_fetcher.time_series.fetch_dataframe([\"x\", \"y\", \"mean_x_y\"]).plot()"
```

Comment:
Don't need/want `x="timestamp"` anymore?

Reply:
As above, not anymore.